src/TropicalGeometry/linear_space.jl

```diff
@@ -365,6 +365,30 @@ function tropical_linear_space(I::MPolyIdeal, nu::Union{Nothing,TropicalSemiring
 end
 
 
+@doc raw"""
+    tropical_linear_space(G::Graph, nu::TropicalSemiringMap; weighted_polyhedral_complex_only::Bool=false)
```

Comment:
```suggestion
    tropical_linear_space(G::Graph[, nu::TropicalSemiringMap]; weighted_polyhedral_complex_only::Bool=false)
```
Maybe mention that this is optional (and what it defaults to)?

Reply:
I think that the design is currently not really consistent.

The [doc intro](https://github.com/oscar-system/Oscar.jl/blob/master/docs/src/TropicalGeometry/intro.md) says "Like [[MS15](https://docs.oscar-system.org/dev/references/#MS15)], OSCAR follows the min-convention by default and unless specified otherwise."  Yet the code, e.g., [here](https://github.com/oscar-system/Oscar.jl/blob/d9efcbce878eb3bad9c3b220edd42b723c0add75/src/TropicalGeometry/initial.jl#L25) does not set any default.